modules/users/server/models/user.server.model.js

```diff
@@ -112,7 +112,7 @@ UserSchema.pre('save', function(next) {
  */
 UserSchema.methods.hashPassword = function(password) {
 	if (this.salt && password) {
-		return crypto.pbkdf2Sync(password, this.salt, 10000, 64).toString('base64');
+		return crypto.pbkdf2Sync(password, new Buffer(this.salt, 'base64'), 10000, 64).toString('base64');
```

Comment:
Re-instantiating the buffer seems unnecessary when we could pass in L183 without the `.toString('base64')`, no?